atom

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# This script is called 'atom' to surpass the react-native
+# editor selection. It has nothing to do with the atom editor.
+# It can be safetly removed when the PR to support vscode in
+# react native gets accepted:
+#
+#
+# Usage:
+# ../path/atom filename:filenumber
+
+if [[ -z "$1" ]] ; then
+    echo 'Missing filename.'
+    exit 1
+fi
+
+parent_path=$( cd "$(dirname "${BASH_SOURCE}")" ; pwd -P )
+node "$parent_path/out/extension/openFileAtLocation.js" $1
```

Comment:
Does this `$1` need to be quoted as well in the case of paths with spaces in them?